README.md

```diff
@@ -1,122 +1,214 @@
-_This document is a work in progress, please make PRs_ _What matters at this
```

Comment:
This disclaimer should be kept.

Reply:
Ah, merged it with the other disclaimer to create one, [here](https://github.com/atomone-hub/genesis/pull/92/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R14-R17).

Should it be kept separate?

Reply:
The README should include a disclaimer and serve as a self-contained document. It should also contain a todo list. This way, regardless of the version, you can read the README and find the disclaimer (if applicable), the current content, and the todo list.

Reply:
@moul addressed.